### PR TITLE
🎨 Roles update

### PR DESCRIPTION
**Pull request by jsaq007**
## What do these changes do?
@esraneufeld could you let me know if I got missed something with the renaming for this as the topic wasn't very clear having not been part of the discussion with @odeimaiz 

Separate the resource for, studies and template and new logic for Services
- Owner
- Editor (collaborator )*
- User (viewer)*

Services
- Owner
- Editor (collaborator )*
- User (viewer)*

Organisation
- Restricted member (user)*
- Member
- Manager
- Administrator

*new role (changed role)

Roles-update 2023-11-01 11-25

### CODE REVIEW

**Review by GitHK — APPROVED** [review]
👍

File: services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js [hunk 1]
```diff
@@ -1107,6 +1107,7 @@ qx.Class.define("osparc.dashboard.StudyBrowser", {
 
       let operationPromise = null;
       if (collabGids.length > 1 && amICollaborator) {
+        // Fixme - Study collaborators should be users.
```
Comment (pcrespov) [inline services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js:1110]: IN the backend we avoid leaving FIXME and TODO comments. SEE [docs/coding-convensions.md -> CC1](https://github.com/ITISFoundation/osparc-simcore/blob/68a425c36f565acfdc22c1f00000b8183041e2e0/docs/coding-conventions.md#cc1-can-i-use-todo-fixme) Our PR review bot from sonar detects them (i forget most of the time to go through them). 
I would advice to follow the same policy but I most importantly is to agree with @ignapas about this policy.

**Review by matusdrobuliak66 — APPROVED** [review]
:+1:

**Review by sanderegg — APPROVED** [review]

**Review by elisabettai — APPROVED** [review]
Thanks! 

Small suggestion for the PR description: it is not clear which roles have been renamed (you have the "*" at the bottom, but you don't use it above). Maybe you can just write next to them "Changed: change reason"

File: services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js (hunk 1 above)
Comment (jsaq007) [reply services/static-webserver/client/source/class/osparc/dashboard/StudyBrowser.js:1110]: I'll amend it :)

**Review by ignapas — APPROVED** [review]
GUI not tested

**Pull request merged**